oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/requests/search/MatchQuery.java

```diff
@@ -38,10 +38,14 @@
         public void serialize(final MatchQuery value, final JsonGenerator gen,
                               final SerializerProvider provider)
             throws IOException {
-            gen.writeFieldName("match");
             gen.writeStartObject();
             {
-                gen.writeStringField(value.getName(), value.getText());
+                gen.writeFieldName("match");
+                gen.writeStartObject();
+                {
```

Comment:
What are these brackets for?

Reply:
Just a code style?

Reply:
> Just a code style?

Yes, this is a kind of code style to make it easier to distinguish the written object structures, and help to write the correct `writeStartXxxx` / `writeEndXxx`, otherwise I have to count the number of `writeStartObject` / `writeEndObject`